Packs/CommonScripts/ReleaseNotes/1_19_43.md

```diff
@@ -0,0 +1,13 @@
+
+#### Scripts
+
+##### clear-user-session
```

Comment:
Fix the file so it looks as it should. 
I think there are too many spaces before the '-'
Also no need for so many blank lines at the end :)